src/components/participants-list/participants-list.tsx

```diff
@@ -0,0 +1,43 @@
+import { Participant } from "@/app/events/[event]/participants/page";
```

Comment:
If this type will be shared, we should move it to a common/shared folder.

Reply:
Ok. The types that i created may change, i did it based on [layout](https://www.figma.com/file/PsD124B5jvDdxyYCqxIbys/Code-in-The-Dark---Back-Office?type=design&node-id=633-5459&mode=dev) fields.